packages/frameworks/src/frameworks.ts

```diff
@@ -1666,7 +1617,6 @@ export const frameworks = [
       },
     },
     dependency: 'nuxt',
-    getFsOutputDir: async () => '.output',
```

Comment:
Ditto.

Reply:
Since `.output` is the default value, it can be omitted like the PR desc mentions

Reply:
If the framework emits `.output` then it will never call the `getFsOutputDir()` function because of this `if` statement:

https://github.com/vercel/vercel/blob/9c1b7dca31c8e51e8eda6a1aa6801dd72cb8e57d/packages/cli/src/commands/build.ts#L336

Reply:
I see, thanks. Disregard this comment then.